.github/workflows/endpoint-test.yml

```diff
@@ -30,6 +30,10 @@ jobs:
       - name: Log into Azure
         uses: azure/login@v2
         with:
+          # Use a managed identity to authenticate to Azure
+          # Use properties such as client-id, tenant-id, and subscription-id in the secrets and vars
+          # The managed identity should have a federated credential with subject identiier repo:<organisation>/<repo>:pull_request
+          # Use subject identiier repo:<organisation>/<repo>:ref:refs/heads/<branch> for running manual CI's
```

Comment:
I don't think the word repo makes sense here? It's just a subject identifier

Reply:
for creating the federated credential you need to enter the organization and repo to produce the subject identifier. I believe it is good to clarify that.

Reply:
Oh sorry you're right the line wrapping on my view made it look like repo wasn't part of the identifier but of course it is